web/libs/ui/src/tailwind.config.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
         black: "#181C31",
         blackho: "#2C3149",
         blacksection: "#1C2136",
-        primaryho: "#0063EC",
+        primaryho: "#37447A",
```

Comment:
Can we change these shortened forms to something else, it is in english forming a derogatory term. What is `ho` short for in these color names?

Reply:
it's shadcn thing. `ho` is `hover`

Reply:
I've never noticed that before, but that is a pretty glaring oversight on their part. As we already discussed, we will look to remove that problem with design token alignment process with @ricardoantoniocm 👍